test/backend_test.in.cpp

```diff
@@ -2687,6 +2687,12 @@ NGRAPH_TEST(${BACKEND_NAME}, zero_sized_ceiling)
     make_unary_empty_test<op::Ceiling>("${BACKEND_NAME}");
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, zero_sized_erf)
+{
+    // TODO: Fix when backends are ready
+    make_unary_empty_test<op::Erf>("INTERPRETER");
```

Comment:
Instead of hard-coding to `INTERPRETER` we should put `${BACKEND_NAME}` and disable the test in the other backends' manifests.